src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -143,18 +145,31 @@ private static async Task<string> SanitizeRequestBodyAsync(Stream requestStream)
                 return "Request body could not be tracked because stream is not seekable";
             }
 
-            long originalPosition = requestStream.Position;
-            if (requestStream.Position != 0)
+            string contents = null;
+            long? originalPosition = null;
+
+            try
             {
-                requestStream.Seek(0, SeekOrigin.Begin);
-            }
+                originalPosition = requestStream.Position;
+                if (requestStream.Position != 0)
+                {
+                    requestStream.Seek(0, SeekOrigin.Begin);
+                }
 
-            var reader = new StreamReader(requestStream);
-            string contents = await reader.ReadToEndAsync();
+                var reader = new StreamReader(requestStream);
+                contents = await reader.ReadToEndAsync();
+            }
+            catch (Exception ex)
+            {
+                _logger.LogWarning(ex, "Unable to get request body for request tracking");
```

Comment:
```suggestion
            catch (Exception exception)
            {
                _logger.LogWarning(exception, "Unable to get request body for request tracking");
```

Something we do in Arcus, right? Full names except for lamdba's. 😉

Reply:
I don't like abbreviations :D